install.py

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env python
 
-# Copyright 2018 The NATS Authors
+# Copyright 2018,2020 The NATS Authors
```

Comment:
2018-2020 is style we have been using.

Reply:
fixed

Reply:
I still see the comma version, wanted 2018-2020.

Reply:
You might have been looking at an outdated version, per GitHub's yellow "Outdated" boxes.  The code as landed on master is very clearly `2018-2020`.

Reply:
ok thx